packages/flutter/lib/src/services/image_provider.dart

```diff
@@ -624,6 +629,11 @@ class ExactAssetImage extends AssetBundleImageProvider {
   /// using the key given by [name].
   final AssetBundle bundle;
 
+  /// The name of the package from which the image is included.
+  ///
+  /// This must be null when the image is included by the current app.
```

Comment:
this should include more details about:
(a) what this does
(b) how you write `pubspec.yaml` files to use this